configs/client_cli/config.json

```
@@ -6,6 +6,8 @@
       "name": "wonderland"
     }
   },
+  "LOGIN": "mad_hatter",
```

```suggestion
  "WEB_LOGIN": "mad_hatter",
```

I'd add a comment here explaining the use case. Since most people can confuse their `LOGIN` with the `ACCOUNT` that they'll use with the blockchain, I think an explicit prefix of `WEB` makes sense here.

---

I don't see any conflicts. You need to ensure that the user knows that this is not used for blockchain operations, just to get through basic authentication "firewalls". That comment is about something completely different.

---

Done in new commit